releases/release-1.13/release_team.md

```diff
@@ -3,11 +3,11 @@
 | Lead | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) | Aaron Crickenberger ([@spiffxp](https://github.com/spiffxp)) |
 | Enhancements | Kendrick Coleman ([@kacole2](https://github.com/kacole2)) | Michael Singh ([@wadadli](https://github.com/wadadli)), Guinevere Saenger ([@guineveresaenger](https://github.com/guineveresaenger) / @gsaenger on Slack), Arnaud Meukam ([@ameukam](https://github.com/ameukam)), Claire Laurence ([@claurence](https://github.com/claurence)) |
 | CI Signal | Josh Berkus ([@jberkus](https://github.com/jberkus)) | Shivaani Mall ([@shivani1494](https://github.com/shivani1494)), Morten Torkildsen ([@mortent](https://github.com/mortent)), Tao Beier ([@tao12345666333](https://github.com/tao12345666333)), Maria Ntalla ([@mariantalla](https://github.com/mariantalla)) |
-| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)), Dhawal Yogesh Bhanushali ([@imkin](https://github.com/imkin)) |
-| Bug Triage | Niko Penteridis ([@nikopen](https://github.com/nikopen)) | Mohammed Zeeshan Ahmed ([@mohammedzee1000](https://github.com/mohammedzee1000)), Athanasios Kostopoulos ([@thanasisk](https://github.com/thanasisk)), Shubheksha Jalan ([@shubheksha](https://github.com/shubheksha)) |
+| Test Infra | Cole Wagner ([@cjwagner](https://github.com/cjwagner)) | Amit Watve ([@amwat](https://github.com/amwat)), Dhawal Yogesh Bhanushali ([@imkin](https://github.com/imkin)), Conrad Taylor ([@conradwt](https://github.com/conradwt)), Rahul Mahale ([@RahulMahale](https://github.com/RahulMahale)) |
+| Bug Triage | Niko Penteridis ([@nikopen](https://github.com/nikopen)) | Mohammed Zeeshan Ahmed ([@mohammedzee1000](https://github.com/mohammedzee1000)), Athanasios Kostopoulos ([@thanasisk](https://github.com/thanasisk)), Shubheksha Jalan ([@shubheksha](https://github.com/shubheksha)), Tunde ([@simplytunde](https://github.com/simplytunde)) |
```

Comment:
As per guidance in the SIG Release meeting, we decided to cap the shadows to 4 per role. So probably we can add Tunde as an official shadow for 1.14? But feel free to join the 1.13 meetings if you are interested @simplytunde

Reply:
Yep. It's capped at four here, including Tunde.